components/portal/timeout/controllers.js

```diff
@@ -103,7 +103,7 @@ define(['angular'], function(angular) {
      * Redirect to end the session;
      */
     function redirect() {
-      $window.location.replace(MISC_URLS.loginURL);
```

Comment:
Handle case where logoutURL not configured, or figure it's always configured and so can be relied upon?